fcrepo4/fcrepo4
# Fcrepo 2723: If acl URI exists but does not contain the correct triple, return 409

**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2723

This ticket completes work originally described in https://jira.duraspace.org/browse/FCREPO-2705

# What does this Pull Request do?
It adds logic to check that an acl uri, if it exists, has been given the RDF:type of webac:Acl; if it has not, then a 409 (Conflict) code is returned along with a descriptive error message.

# What's new?
- updates FedoraLdp to return 409 in case acl resource doesn’t have
type web:Acl

- updates private createAcl() method in FedoraLdpIT to add the
required type to an acl and adds second private method to test for
cases where type is not present

- adds unique id to aclPid for each test

- adds slug to new POST tests for consistency

Resolves: https://jira.duraspace.org/browse/FCREPO-2723

# How should this be tested?

There are two new IT test cases in FedoraLdpIT that check to see that a 409 is returned when an acl exists but is not correctly typed with webac:Acl and four existing test cases that should return 201 because they have the webac:Acl type.

# Interested parties
@dbernstein @awoods
